css/styles.css

```diff
@@ -65,7 +65,7 @@ strong {
   display: inline-block;
   padding: 0.5rem 1.5rem;
   transform: rotate(-16deg);
-  letter-spacing: 0.3rem;
+  letter-spacing: 0.0rem;
```

Comment:
This `letter-spacing` is intended by design. Do you think the arabic language shouldn't have it?

Reply:
@sergeicodes 

> Do you think the arabic language shouldn't have it?

yes, its make the sentence no readability in Arabic.

Reply:
Then could you please revert your code in `css/styles.css` and add your override to `ar/index.html` to the `<style>` tag in the `<head>`:

```css
:lang(ar) .stamp { {
  letter-spacing: 0;
}
```

Reply:
> Hmmm, I don't see any changes
> ![image](https://user-images.githubusercontent.com/10166916/177184761-a4f71cf1-79a5-4342-93f4-2582bf642419.png)

Sorry, I make new pull request now, please check it. And tell me if everything fine